sea-query-derive/tests/pass/flattened_unnamed.rs

```diff
@@ -28,7 +28,9 @@ fn main() {
     let expected = ["asset", "id", "asset_name", "user_id", "creation_date"];
     Asset::iter()
         .chain(std::iter::once(Asset::Creation(CreationInfo::Date)))
-        .map(|var| Iden::to_string(&var))
         .zip(expected)
-        .for_each(|(iden, exp)| assert_eq!(iden, exp))
+        .for_each(|(var, exp)| {
+            assert_eq!(Iden::to_string(&var), exp);
```

Comment:
Why do you write it like that and not `var.to_string()` and `var.as_str()`?

Reply:
Done